modules/lightning-charge.nix

```diff
@@ -24,23 +29,29 @@ in {
     };
     users.groups.lightning-charge = {};
 
+
+    systemd.tmpfiles.rules = [
+      "d '${cfg.dataDir}' 0700 ${config.users.users.lightning-charge.name} ${config.users.users.lightning-charge.group} - -"
+    ];
+
     environment.systemPackages = [ pkgs.nix-bitcoin.lightning-charge ];
     systemd.services.lightning-charge = {
       description = "Run lightning-charge";
       wantedBy = [ "multi-user.target" ];
       requires = [ "clightning.service" ];
       after = [ "clightning.service" ];
       preStart = ''
-        # Give existing lightning-charge.db right permissions
+        # Move existing lightning-charge.db
```

Comment:
Perhaps add a TODO that this can be removed eventually.

Reply:
Resolved in 1165fc6